rpcserver.go

```diff
@@ -7148,7 +7148,9 @@ func (r *rpcServer) VerifyChanBackup(ctx context.Context,
 		}
 	}
 
-	return &lnrpc.VerifyChanBackupResponse{}, nil
+	return &lnrpc.VerifyChanBackupResponse{
+		Success: true,
```

Comment:
Can this ever be false?

Reply:
No, this is purely to improve the output on the command line (which currently is just empty braces).